website/src/lib/wordpress.js

```diff
@@ -90,36 +91,78 @@ let fragmentCategories = /* GraphQL */ `
   }
 `;
 
-export async function getPreviewPost(id, idType = 'DATABASE_ID') {
-  const data = await fetchAPI(
-    `
-    query PreviewPost($id: ID!, $idType: PostIdType!) {
-      post(id: $id, idType: $idType) {
-        databaseId
-        slug
+export async function getContentTypes() {
```

Comment:
My original thought was that we'd add this to `getGlobalProps`, so we have it in our globals context. But if it's not useful anywhere other than here, that's not needed.

Reply:
I don't see a need for this anywhere else on the site and it actually requires authentication to get the rest path value. We don't want to add authenticated wordpress queries as a prereq for normal site functionality. The `uri` value we get from posts does everything this does, we just don't have access to that data from preview posts, which is what getContentTypes is doing for us.